shell/config/router/routes.js

```diff
@@ -192,25 +197,27 @@ export default [
         component: () => interopDefault(import('@shell/pages/c/_cluster/explorer/index.vue')),
         name:      'c-cluster-explorer'
       }, {
-        path:      '/c/:cluster/backup',
-        component: () => interopDefault(import('@shell/pages/c/_cluster/backup/index.vue')),
-        name:      'c-cluster-backup'
+        path: '/c/:cluster/backup',
+        name: 'c-cluster-backup',
+        meta: { ...installRedirectRouteMeta(BACKUP_NAME, BACKUP_CHART_NAME, BACKUP_RESTORE.BACKUP) }
```

Comment:
I first attempted this as a `beforeEnter()` route guard but it didn't pan out because we didn't have all of the store data loaded before we got to this point.

I also attempted to use `beforeRouteEnter` in our components but the the guard wouldn't get called I believe before we would redirect.

I don't particularly love this solution and I think we should consider getting rid of this concept entirely but for the time being I just want to preserve existing behavior.